src/ontodia/customization/templates/bigIconOwl.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { Component } from 'react';
+
+import { CrossOriginImage } from '../../viewUtils/crossOriginImage';
+
+import { TemplateProps } from '../props';
+
+const CLASS_NAME = 'ontodia-big-icon-owl_template';
+
+export class BigIconTemplateOwl extends Component<TemplateProps, {}> {
```

Comment:
Since you're providing templates for OWL/VOWL notation please name templates and files appropriately. For example, VOWLClassTemplate, VOWLPropertyTemplate, etc.

Reply:
fixed